com.valvesoftware.Steam.CompatibilityTool.Proton.yml

```
@@ -732,6 +746,53 @@ modules:
         sha256: 8ff2fcf9b054e4a4ef56fcd8a637322f827b2b176a592a618d63672ddb896e06
 
 
+  - name: vkd3d
+    subdir: vkd3d
+    build-options:
+      ldflags: >-
+        -L/usr/lib/sdk/mingw-w64/x86_64-w64-mingw32/lib
+        -L/app/share/steam/compatibilitytools.d/Proton-GE/lib/wine
```

Probably should be `lib/wine/x86_64-windows` here
```suggestion
        -L/app/share/steam/compatibilitytools.d/Proton-GE/lib/wine/x86_64-windows
```